{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"pymongo.errors.OperationFailure: Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1727228184, 1), 'signature': {'hash': b'\\xbe\\xb2\\xf5S\\x19\\xca\\xabxf\\xa1\\x92 _\\x11\\x97J?\\x99\\xc7\\n', 'keyId': 7418387111310524441}}, 'operationTime': Timestamp(1727228184, 1)}","children":[{"name":"test_mongos_external_connections","uid":"692e050b32cf0e2b","parentUid":"7d1dde6ed8d3f4525568d57c5b660494","status":"broken","time":{"start":1727228166851,"stop":1727228184881,"duration":18030},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":true,"parameters":[],"tags":["asyncio","abort_on_fail","group(1)"]}],"uid":"7d1dde6ed8d3f4525568d57c5b660494"}],"uid":"bdbf199525818fae7a8651db9eafe741"},{"name":"Product defects","children":[{"name":"AssertionError: Mongos is not currently running.\nassert False","children":[{"name":"test_mongos_has_user","uid":"aae966d67f60358f","parentUid":"47316889e8fa06a3c4dea1c03cee6b4f","status":"failed","time":{"start":1727228233199,"stop":1727228243556,"duration":10357},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":true,"parameters":[],"tags":["asyncio","abort_on_fail","group(1)"]}],"uid":"47316889e8fa06a3c4dea1c03cee6b4f"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}